TYPESCRIPT.md

```diff
@@ -92,23 +92,26 @@ Dynamic query keys are based on the given shape of the expected entries' content
 
 ![](images/dynamic-query-keys.png)
 
-To calculate dynamic keys, we have to define the shape of the fields of the entries' content type:
+To calculate dynamic keys, we have to define the shape the entries' content type:
```

Comment:
Combining "shape" and "content type" is somewhat duplicated. Either "we have to provide the shape of the entries:" or "we have to provide the relevant information from the content type:" would work for me.